lib/widget/test_chart.dart

```diff
@@ -0,0 +1,200 @@
+                      RadialAxis(
+                          axisLineStyle: AxisLineStyle(
+                            thickness: 20,
+                          ),
+                          startAngle: -90,
+                          endAngle: 270,
+                          showTicks: false,
+                          showLabels: false,
+                          pointers: <GaugePointer>[
+                            RangePointer(
+                                value: 25, width: 20, color: Color(0xFFC37447))
+                          ],
+                          annotations: <GaugeAnnotation>[
+                            GaugeAnnotation(
+                                widget: Container(
+                                    child: Text('25 %',
+                                        style: TextStyle(
+                                            fontWeight: FontWeight.bold,
+                                            fontSize: 16))),
+                                angle: 90,
+                                positionFactor: 0.2)
+                          ]),
+                    ],
+                  ),
+                )
+              ],
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Container(
+                  width: MediaQuery.of(context).size.width / 2,
+                  child: Text('การอาบน้ำ',
+                      style: Theme.of(context).textTheme.bodyText1,
+                      textAlign: TextAlign.left),
+                ),
+                Container(
+                  height: 120,
+                  width: 100,
+                  child: SfRadialGauge(
+                    axes: <RadialAxis>[
+                      RadialAxis(
+                          axisLineStyle: AxisLineStyle(
+                            thickness: 20,
+                          ),
+                          startAngle: -90,
+                          endAngle: 270,
+                          showTicks: false,
+                          showLabels: false,
+                          pointers: <GaugePointer>[
+                            RangePointer(
+                                value: 50, width: 20, color: Color(0xFFC37447))
+                          ],
+                          annotations: <GaugeAnnotation>[
+                            GaugeAnnotation(
+                                widget: Container(
+                                    child: Text('50 %',
+                                        style: TextStyle(
+                                            fontWeight: FontWeight.bold,
+                                            fontSize: 16))),
+                                angle: 90,
+                                positionFactor: 0.2)
+                          ]),
+                    ],
+                  ),
+                )
+              ],
+            ),
+            //for (var i in [1]) buildItem("Item ${i}"),
```

Comment:
this as well

Reply:
I'm waiting for today's consultation as well.